api_tests/tests/orderbook.ts

```diff
@@ -79,6 +80,8 @@ it(
         const aliceSwapResponse = await alice.cnd.client.get(
             aliceTakeOrderResponse.headers.location
         );
+        console.log(aliceTakeOrderResponse.headers.location);
```

Comment:
Please remove this before you do the final PR, tests should be quiet when they are pass - console.log makes them noisy.

Reply:
Usually, having assertions in the middle of a test is not that great but within e2e tests, it can be useful to do so to catch errors early. Hence, if you care about the `location` header here, you could just assert against it :)

Even `toBeDefined` can be a useful assertion.